src/ebmc/ebmc_parse_options.cpp

```diff
@@ -45,191 +47,197 @@ Function: ebmc_parse_optionst::doit
 
 int ebmc_parse_optionst::doit()
 {
-  if (config.set(cmdline)) {
-    usage_error();
-    exit(CPROVER_EXIT_USAGE_ERROR);
-  }
-  register_languages();
-
-  if(cmdline.isset("version"))
+  try
   {
```

Comment:
Isn't starting the `try` right here taking things a little too far? There is quite a bit of code that I wouldn't expect to throw.

Reply:
moved